tdrs-frontend/src/actions/createXLSReport.js

```diff
@@ -9,24 +9,8 @@ const BACKEND_URL = process.env.REACT_APP_BACKEND_URL
 Get a list of parse errors for a given file id from the backend using the
 `/parsing/parse_errors/{id}` endpoint.
 */
-export const getParseErrors = () => async (dispatch) => {
-  dispatch({
-    type: FETCH_PARSE_ERRORS,
-  })
+export const getParseErrors = (data_json) => {
   try {
-    const response = await axios.get(
```

Comment:
removed this part because we already query the error and have the xlsReport available so this was redundant query